### ISSUE TITLE
Analytics module bug on error pages

### ISSUE THREAD

**Issue by robyedlin** [nuxt-community/analytics-module#3]
On 404 pages (using `layouts/error.vue`) I'm seeing the following in the console:

```
vue-router.esm.js:1832 TypeError: Cannot read property 'components' of undefined
    at i (google-analytics.plugin.76c7bae6.js:33)
    at google-analytics.plugin.76c7bae6.js:20
    at vue-router.esm.js:1926
    at Array.forEach (<anonymous>)
    at e.te.updateRoute (vue-router.esm.js:1925)
    at vue-router.esm.js:1802
    at vue-router.esm.js:1911
    at r (vue-router.esm.js:1649)
    at at (vue-router.esm.js:1660)
    at vue-router.esm.js:1906
```

**Comment by pi0** [comment]
Thanks, @robyedlin for the report. We will try to investigate this issue soon.

**Comment by robyedlin** [comment]
No problem. Thanks for looking into it @pi0

**Comment by robyedlin** [comment]
Hey @pi0, any luck on this? I know you are super busy with 1.0. No rush.

**Comment by awronski** [comment]
@pi0 What's the status of this module, is it production ready?

Thanks for info!

**Comment by robyedlin** [comment]
Any update @pi0

**Comment by robyedlin** [comment]
Fixed with 2.0.2. Thanks @pi0
